test/integration/default/controls/audit_spec.rb

```diff
@@ -14,7 +14,8 @@
 
 # Grab bundled inspec version. This should be the same as the one
 # passed for audit cookbook. If its not, you should do a `bundle install`
-inspec_version = Gem.loaded_specs['inspec'].version.to_s
+inspec_version = Inspec::VERSION
+# or: Gem.loaded_specs['inspec'].version.to_s rescue Inspec::VERSION
```

Comment:
Could you explain this change?

Reply:
The active code. That is using our VERSION constant instead of finding the same value indirectly all the way through rubygems and back. If the spec isn't loaded, then that will fail out. There's no reason that the spec needs to be loaded for this test to run or pass.